drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -173,20 +173,13 @@ union lis2dh_sample {
 	} __packed;
 };
 
-#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
-struct lis2dh_spi_cfg {
-	struct spi_config spi_conf;
-	const char *cs_gpios_label;
-};
-#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(spi) */
-
 union lis2dh_bus_cfg {
 #if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
 	uint16_t i2c_slv_addr;
 #endif
 
 #if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
-	const struct lis2dh_spi_cfg *spi_cfg;
+	struct spi_dt_spec spi;
```

Comment:
Did you drop 'const' on purpose?

Reply:
@avisconti it's no longer a pointer

Reply:
> @avisconti it's no longer a pointer

Yes, correct. But the structure is constant indeed and cannot be changed at runtime (only assigned at compile time from DT). Isn't it?

Reply:
but the whole driver config is already `const`, right?

Reply:
yes, right

Reply:
Sorry for my late answer. @gmarull explained that quite :)